### DISCUSSION TITLE
Any book recommendations for learning Blockly ?

### DISCUSSION THREAD

**Discussion by Cameratrapsue** [gopro/labs#842] [General]
HI, I'm new to coding and blockly. Any recommendations for a book for a complete beginner preferably via Amazon/Kindle.
I've found a book for kids (probably where I'm at learning wise with code) called "Learning Python with Google Blockly: A kids introduction to python programming" would this be suitable to apply to Gopro blockly?
Thanks, Sue

**Comment by dnewman-gpsw** [comment]
That is not likely much help.  Labs uses a very simple language, but very different than Python.  Blockly is just blocks that fit together when appropriate.  Best is to just play.

**Comment by Cameratrapsue** [comment]
Thanks for the quick reply - much appreciated. I'll play around with Blockly and try to learn it that way.
